packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -195,17 +202,15 @@ const SquadPage = ({
         <Feed
           className="px-6 pt-14 laptop:pt-10"
           feedName={OtherFeedPage.Squad}
-          feedQueryKey={[
-            'sourceFeed',
-            user?.id ?? 'anonymous',
-            Object.values(queryVariables),
-          ]}
+          feedQueryKey={feedQueryKey}
```

Comment:
This change is not needed anymore, so if you can adjust as it was to avoid unnecessary diff

Reply:
> This change is not needed anymore, so if you can revert as to avoid unnecessary diff

but then how will I be able to gloat about how many lines of changes my PRs have? 😄

Reply:
😆 touché!